### PR TITLE
Add Permission, OldPermission to AuditEntry type

### PR DESCRIPTION
**Pull request by Archetypically**
Fixes https://github.com/google/go-github/issues/2669

# Changes

- Add marshalling of the `old_permission` and `permission` fields in the response of `GET /orgs/:org/audit-log` to the `AuditEntry` type as `AuditEntry.OldPermission` and `AuditEntry.Permission`.

### CODE REVIEW

**Comment by Archetypically** [comment]
> Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> View this [failed invocation](https://github.com/google/go-github/pull/2670/checks?check_run_id=11530525237) of the CLA check for more information.
> 
> For the most up to date status, view the checks section at the bottom of the pull request.

Signed CLA.

File: github/orgs_audit_log_test.go [hunk 1]
```diff
@@ -34,6 +34,8 @@ func TestOrganizationService_GetAuditLog(t *testing.T) {
         "created_at": 1615077308538,
         "head_sha": "5acdeadbeef64d1a62388e901e5cdc9358644b37",
         "conclusion": "success",
+		"old_permission": "read",
+		"permission": "admin",
```
Comment (gmlewis) [inline github/orgs_audit_log_test.go:38]: Nit: Would you mind making these two lines use the same spacing as lines 34-36 above them?

Comment (Archetypically) [reply github/orgs_audit_log_test.go:38]: Just to clarify; you're referring to the use of spaces instead of tabs?

Comment (gmlewis) [reply github/orgs_audit_log_test.go:38]: Yes, exactly... sorry, and thank you!

Comment (Archetypically) [reply github/orgs_audit_log_test.go:38]: No worries at all; will do!

**Review by gmlewis — APPROVED** [review]
Thank you, @Archetypically !
LGTM.
Merging after tests pass.

**Comment by Archetypically** [comment]
Thanks @gmlewis!

**Pull request merged**